about/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>About me</title>
+</head>
+<body>
+    <h1>Обо мне</h1>
+    <h2>Привет!</h2>
+    <h3>Меня зовут Александр! Мне 36 лет. Живу в Минске.</h3>
+    <hr/>
+    <p>По специальности я <mark>инженер</mark> в автомобильной сфере. Работаю в этой области <ins>более <sub>1</sub><sup>0</sup> лет</ins>.
+        <br/>Но порой может наступить момент, когда перестаешь получать удовлетворение от проделанной работы и как говоpится <q>начинаешь искать новый вызов</q>.
+    </p>
+    <p>Ещё до окончания университета, будучи студентом, я подрабатывал в интернет-провайдере. Это было ещё на заре пирингового интернета. 
+        <br/>Видел как менялись веб-страницы разных компаний. Как от <em>сайтов-визиток</em> всё дошло до <strong>веб-приложений</strong>. 
+        <br/>Мне всегда <del>было интересно</del> хотелось узнать с технической точки зрения как это реализовано.
+    </p>
```

Comment:
It is bad practice to use `<br/>` in the paragraph. If need separate text, create a new paragraph.

Reply:
Fix this problem

Reply:
Paragraph in paragraph looks wrong, what do you think?

Reply:
I think this is my mistake. Fix it.